lib/backpex/fields/has_many_through.ex

```diff
@@ -359,10 +355,10 @@ defmodule Backpex.Fields.HasManyThrough do
     existing = get_change_or_field(changeset, association.pivot.field)
     {to_delete, rest} = List.pop_at(existing, index)
 
-    primary_key_field = EctoUtils.get_primary_key_field(to_delete)
+    primary_key = socket.assigns.live_resource.config(:primary_key)
```

Comment:
We might use the wrong primary key here. This has to be the primary key of the associated resource.

Reply:
Done.